src/TodoList/TodoList.tsx

```diff
@@ -52,11 +52,12 @@ const TodoList:FC<TodoListProps> = (props: TodoListProps) => {
                 visibleItems.map((item: Element) => {
                     return(
                         <li key={item.id} className={s.element}>
-                            <span onClick={(e) => setDone(e, item.id)} >
+                            <span>
                                 <input 
+                                onClick={(e) => setDone(e, item.id)}
                                 id={item.id.toString()} 
-                                className={s.custom_radio} 
-                                type="radio" 
+                                className={s.custom_checkbox} 
+                                type="checkbox"
                                 checked={item.done} 
                                 onChange={()=>console.log('')}/>
```

Comment:
I would propose to use`onChange` callback instead of `onClick`;
and also consider using `e.target.checked` to get Item state from the checkbox